src/main/resources/lib/channels.fjs

```diff
@@ -11,7 +11,7 @@ type Channel<A> = { put         : (A) => PutResult<A>
                   , apply       : <B>(Channel<(A) => B>) => Channel<B>
                   , merge       : (Channel<A>) => Channel<A>
                   , dropRepeats : (([A,A]) => boolean) => Channel<A>
-                  , takeN       : (number) => Channel<A>
```

Comment:
why is that?

Reply:
It throws an error. This is a hacky fix, I'm sure there is a better way. Will come up now to show you